src/components/status/list/content.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState, useEffect } from 'react';
+import classnames from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+interface IContent {
+  index: number;
+  nowIndex: number;
+  changeIndex: (display: number) => void;
+  bidType: string;
+  status: string;
+  date: string;
+  time: string;
+  price: number;
+  hash: string;
+}
+
+const Content: React.FC<IContent> = ({
+  index,
+  nowIndex,
+  changeIndex,
+  bidType,
+  status,
+  date,
+  time,
+  price,
+  hash,
+}) => {
+  const { t } = useTranslation();
+
+  // change button color and pass state
+  const button = classnames('status-list-content-button');
+  const buttonClick = classnames(
+    'status-list-content-button',
+    'status-list-content-button--click',
+  );
+  const [buttonColor, setButtonColor] = useState<string>(button);
+  const [color, setColor] = useState<boolean>(false);
+  useEffect(() => {
+    setButtonColor(color === false ? button : buttonClick);
```

Comment:
Since this variable is boolean, you can just simply do this `setButtonColor(color ? button : buttonClick)`.
And so you can see named it `color` is very difficult to understand what the variable is mean.

Reply:
done